sentry/src/test/java/io/sentry/protocol/UserSerializationTest.kt

```diff
@@ -43,6 +43,15 @@ class UserSerializationTest {
         assertEquals(expectedJson, actualJson)
     }
 
+    @Test
+    fun deserializeLegacy() {
```

Comment:
You can use backticks and call it `deserialize legacy`

Reply:
updated